Steam Web Integration.user.js

```diff
@@ -29,6 +29,7 @@ const prefix = false; // Prefix (true) instead of suffix (false) position icon.
 const wantIgnores = true; // Whether (true) or not (false) you want to display an extra icon for ignored (not interested) apps.
 const wantDecommissioned = true; // Wether (true) or not (false) you want to display an extra icon for removed or delisted (decommissioned) apps.
 const wantCards = true; // Whether (true) or not (false) you want to display an extra icon for apps with cards.
+const wantBundles = false; // Whether (true) or not (false) you want to display an extra icon for previously bundled apps.
```

Comment:
false by default because who knows if it works, and *cough* other opinions available in private